appveyor.yml

```diff
@@ -23,47 +27,98 @@ install:
 
 build: off
 
-before_test:
-  - ps: |
-      ruby -v
-      gem -v
-      bundle -v
-      type Gemfile.lock
-      $CACertFile = Join-Path -Path $ENV:AppData -ChildPath 'RubyCACert.pem'
-
-      If (-Not (Test-Path -Path $CACertFile)) {
-        "Downloading CA Cert bundle.."
-          [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
-            Invoke-WebRequest -Uri 'https://curl.haxx.se/ca/cacert.pem' -UseBasicParsing -OutFile $CACertFile | Out-Null
+for:
+  -
+    matrix:
+      exclude:
+        - configuration: Agentless
+        - configuration: Agentfull
+          RUBY_VERSION: 23
+        - configuration: Agentfull
+          RUBY_VERSION: 24
```

Comment:
To avoid massive pile-ups in AppVeyor, I think we should drop Ruby 2.4 support. Ruby 2.3 and 2.5 seems sufficient.

Reply:
Absolutely. Working on that now.